kustomize/base/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
All the airgap Jobs run in this namespace so it's easier to keep track of what's been created.